app.config

```diff
@@ -7,24 +7,29 @@
     <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2"/>
   </startup>
   <appSettings>
-    <add key="UpdateLocationVersion" value="https://firmware.oborne.me/MissionPlanner/upgrade/version.txt"/>
-    <add key="UpdateLocation" value="https://firmware.ardupilot.org/MissionPlanner/upgrade/"/>
-    <add key="UpdateLocationMD5" value="https://firmware.ardupilot.org/MissionPlanner/checksums.txt"/>
-    <add key="UpdateLocationZip" value="https://firmware.ardupilot.org/MissionPlanner/MissionPlanner-latest.zip"/>
-    <add key="BetaUpdateLocationVersion" value="https://github.com/ArduPilot/MissionPlanner/releases/download/betarelease/version.txt"/>

```

Comment:
the issue is is stuff like this is a regression, and shouldnt be in the pull request

Reply:
OK, perhaps I'm best off completely starting again with a new pull request forked anew from master. I'll put that together in parallel to this one.